File: code_examples/4-Functions/3-Return_Values/Go/main.go
/**
 * Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program calculates the area of circle
 */

 package main

 import (
	 "fmt"
	 "math"
 )
 
 func calculateArea(radius int) {
	 // this function calculates the area of circle
 
	 // process
	 area := math.Pi * radius ** 2
 
	return area
 }
 
 func main() {
	var radius int
	var area float

	 // input
	 fmt.Print("Enter the radius of a circle (cm): ")
	 fmt.Scanln(&radius)
	 fmt.Println()
 
	 // call functions
	 area = calculateArea(radius)

	 // output
	 fmt.Printf("The area is %d cm²\n", area)
 
	 fmt.Println("\nDone.")
 }
 

File: code_examples/4-Functions/3-Return_Values/JavaScript/main.js
/**
 * Created by: Mr. Coxall
 * Created on: Sep 2020
 * This program calculates the area of circle
 */

const prompt = require('prompt-sync')();

function calculateArea(radius) {
  // this function calculates the area of circle
  
  // process
  const area = Math.PI * radius ** 2;

  return area
}

// input
const radius = parseInt(prompt("Enter the radius of a circle (cm): "));
console.log();

// call functions
area = calculateArea(radius);

// output
console.log(`The area is ${area} cm²`);

console.log("\nDone.");
